### ISSUE TITLE
FAQ Button and page

### ISSUE THREAD

**Issue by atteggiani** [ACCESS-Hive/access-hive.github.io#493]
## Description
It could be useful to have a FAQ page to explain some frequently asked questions

## Why
Help new users understand core concepts

## Close Criteria
Implement FAQ page (maybe in about/community resources)??
And reference it from the homepage

**Comment by atteggiani** [comment]
@aidanheerdegen  can you please come up with a few important questions to put there? 
:)

**Comment by svenbuder** [comment]
Example Question: How do the different parts of ACCESS-NRI work together with the community?
![Screen Shot 2023-07-28 at 5.24.36 pm.png](https://images.zenhubusercontent.com/6466c5964c9274b41b4b424e/c83b6b67-765f-410a-8547-38395224ac56)

**Comment by svenbuder** [comment]
Example question: What is the difference between TEAMS and WORKING GROUPS?

**Comment by aidanheerdegen** [comment]
# What is the difference between a model component, model configuration and just a model?

A model component is, usually, a single codebase that is typically compiled into a single executable. Strictly speaking a model component is compiled model code that runs independently and communicates with other components via a coupler.

When we say "model", it is a general descriptive name for a particular combination of model components.

For example ACCESS-CM2 has 3 major model components: atmosphere (including land), ocean and sea-ice. By contrast ACCESS-OM2 has 2 major model components: ocean and sea-ice, with the atmospheric forcing provided by a data source.

A model configuration is the model (as described above) with a *specific* configuration of each of the components.

**Comment by aidanheerdegen** [comment]
### What is an experiment? Is this different from a run?

An experiment is when a model configuration is executed, usually on an HPC system. Typically an experiment consists of a number of separate sequential "runs", with each run picking up where the previous run finished. 

So an experiment consists of many discrete separate executions of the model that are linked by passing model state from one run to the next.

**Comment by aidanheerdegen** [comment]
### What is a control experiment? Why are they important?

A "control experiment" (sometimes also called a "control run") is an experiment that is designed to be used as a comparison against which perturbation experiments can be compared.

Control experiments are typically designed to represent some neutral climate state. For example, pre-industrial control experiments are designed to represent a modern climate state without significant anthropogenic climate forcing (global warming).

Control experiments are ideally well equilibrated, so that the model doesn't systematically change (often referred to as "drift").

Control experiments are important because it is an enormous saving in resources if one control experiment can be used by many researchers. It also means researchers can compare their perturbation experiments against not just the control run, but against each other.

**Comment by atteggiani** [comment]
I really like these questions and answers. I think they are very useful especially for beginners to get a quick hold of basic concepts that might be recurring in the website.

I have a few points related to @aidanheerdegen's Q/A:
 
> ### What is the difference between a model component, model configuration and just a model?
> A model component is, usually, a single codebase that is typically compiled into a single executable. Strictly speaking a model component is compiled model code that runs independently and communicates with other components via a coupler.
> 
> When we say "model", it is a general descriptive name for a particular combination of model components.
> 
> For example ACCESS-CM2 has 3 major model components: atmosphere (including land), ocean and sea-ice. By contrast ACCESS-OM2 has 2 major model components: ocean and sea-ice, with the atmospheric forcing provided by a data source.
> 
> A model configuration is the model (as described above) with a _specific_ configuration of each of the components.

I would rephrase this Q/A something like: 
### What is the difference between model, model component, and model configuration?
A _model component_ (sometimes referred to simply as _component_) is a codebase that is typically compiled into a single executable. It usually runs independently and communicates with other _components_ via a coupler.
 
We refer to _model_ as a general descriptive name for a particular combination of _model components_.
For example, ACCESS-CM2 has 3 major _model components_: atmosphere (including land), ocean, and sea-ice. By contrast, ACCESS-OM2 has 2 major _model components_: ocean and sea-ice, with the atmospheric forcing provided by a data source.

A _model configuration_ is a _model_ (as described above) with a specific configuration (e.g. version, set of parameters) for each of its _components_.
<br><br><br>

> ### What is an experiment? Is this different from a run?
> An experiment is when a model configuration is executed, usually on an HPC system. Typically an experiment consists of a > number of separate sequential "runs", with each run picking up where the previous run finished.

> So an experiment consists of many discrete separate executions of the model that are linked by passing model state from > one run to the next.

- There are some experiments that consist even only of a single "run"
I would rephrase this into something like: 
### What is an experiment? Is it different from a run?
A _run_ is when a model configuration is executed, usually on an HPC system. 

An _experiment_ typically consists of one or multiple separate sequential _runs_, with each _run_ picking up where the previous one finished.

So _experiment_ and _run_ can sometimes be used as synonyms, but other times an _experiment_ consists of many _runs_ that are linked by passing model state from one _run_ to the next.
<br><br><br>


- Should we also add a Q/A specifying how do we differentiate experiment from model configuration? To what extent I can perturbate a model configuration to produce different experiments, but still have the same model configuration?

**Comment by aidanheerdegen** [comment]
### What is a perturbation experiment?

A perturbation experiment is when a model configuration from a control experiment is altered in some way. The altered configuration is then run, restarting from a point in the control experiment.

The perturbation experiment is run in the same was as the control, each successive run using the state (restarts) from the previous run. The perturbation experiment can then be compared with the control experiment to see the effect of the perturbation.

Perturbation experiments are a very important way to try and understand complex earth systems, e.g. to isolate potential climate change signals and try and understand their effects.

**Comment by aidanheerdegen** [comment]
> To what extent I can perturbate a model configuration to produce different experiments, but still have the same model configuration?

Heh. We crossed, as I was in the middle of explaining perturbation experiments and had only just come back and completed it.

I think your question is maybe a little more in-depth than a general Q&A. I was even wondering if what I had put down was a bit too much, but I do think some of this isn't obvious to those new to this field.

**Comment by aidanheerdegen** [comment]
I also wondered if we shouldn't explicitly mention the word "restarts" when describing runs/experiments. As the word is used quite often, but perhaps not explained.

**Comment by atteggiani** [comment]
Yeah you're right, my question is too in-depth for a Q&A. 
And I think the basic concepts of what is a perturbation experiment are well captured in your Q/A, maybe I would add a few concrete examples of alterations that might be put in place to be a bit clearer to newbies (e.g. altering atmospheric CO2 concentration levels, altering some parameters in any of the different equations within the model configuration, etc.).

And it's true, maybe it's worth using the word "restart" with a little explanation, since there's no mention to it.

**Comment by aidanheerdegen** [comment]
I provided some FAQs, but I'm assuming someone else is going to make a PR from this? I think that is the best next step, then we can discuss wording of specific Q and As in the PR. I'm happy to participate in a review.

**Comment by atteggiani** [comment]
Yeah, we definitely need to open a PR, and we have to understand how to structure the page and where to put it. 

I think a link to it should definitely be in the homepage, and maybe the page can be accessed from (and effectively placed in) the community resources tab?

For the structure, we might get some inspiration from the pages listed here:
https://blog.hubspot.com/service/faq-page

I think if we don't have too many FAQs (which I think it's the purpose), it's best to keep them all in one single page, one after the other. Similarly to what is done in Wikipedia's FAQ page: https://en.wikipedia.org/wiki/Wikipedia:FAQ/Main

For the page I really like this image (but not sure if we can use it)
https://www.shopify.com/faq

**Comment by aidanheerdegen** [comment]
For the ACCESS-NRI Support FAQ (currently not world readable by all) we went with the content collapsed and only the questions visible

https://forum.access-hive.org.au/t/support-faq-frequently-asked-questions/1021

I really like this approach. It makes it possible to have a large number of questions, but be able to scan them quickly and only select the content (answer) you want to view. 

Here are a couple of examples

https://www.ucas.com/search/frequently-asked-questions/undergraduate

https://www.freespirittravelinsurance.com/faqs/

**Comment by svenbuder** [comment]
**What is the difference between the ACCESS-Hive and the Hive Forum?**

<img width="703" alt="access_hive_links" src="https://github.com/ACCESS-Hive/access-hive.github.io/assets/22073890/7c113968-5d75-4b40-805d-7c251279a9fa">

**Comment by atteggiani** [comment]
@aidanheerdegen 
I really like the collapsed FAQ! 
We can get inspiration by some of those you shared for the page on the Hive. 

@svenbuder 
I like the question and the image to visually show how all fits togethter! 
I would add some text to it as well (we can come up with something).

**Comment by atteggiani** [comment]
Integrate FAQ button in the "Get Support" button on the Homepage --> "FAQs / Get Support"

Integrate FAQ page in the About > User Support page, showing FAQs first, and then the User Support (Forum) in case people still need help.
Connected to #460
